packages/sync-actions/test/product-sync-prices.spec.js

```diff
@@ -664,7 +827,7 @@ describe('Actions', () => {
                 centAmount: 8995,
                 currencyCode: 'EUR',
               },
-              custom: {
+              customField: {
```

Comment:
This looks wrong, please revert this change.

Reply:
done